Add repro for #13332

nemanjaglumac opened the pull request:
### Status
READY

### What does this PR accomplish?
- reproduces #13332 

### How to test this manually?
- `yarn test-cypress-open`
- `frontend/test/metabase/scenarios/question/native.cy.spec.js`
- replace `it.skip()` with `it.only()`
- test should fail until the issue is resolved

### Related Issues
- #3354 
- #13158

### Additional notes:
- I started using `[ci skip]` until code reviews are done and PR is absolutely ready for merge (cc @rlotun )
- this is probably a stylistic preference, but I never know should I include test within `forEach()` loop (if we are testing different versions of the same question), or should test include the loop. If anyone has strong preference towards either, please let me know.

Commit:
Add repro for #13332 [ci skip]

flamber reviewed (commented):
Not sure why you're creating the question twice. You can use the same source question for both filters.
But maybe it's easier to write it that way in Cypress?
It would probably be better to do assertion with Summarize Count, since it should return `1`, but currently returns `0`
![image](https://user-images.githubusercontent.com/1447303/94919762-4b121500-04b5-11eb-9175-11ae713a97e5.png)

nemanjaglumac reviewed (commented):
> Not sure why you're creating the question twice. You can use the same source question for both filters.
> But maybe it's easier to write it that way in Cypress?

Tbh, there is no special reason - just my initial instinct. I could've gone the other way, for sure.

> It would probably be better to do assertion with Summarize Count, since it should return `1`, but currently returns `0`

Good point. Thank you.

flamber commented on frontend/test/metabase/scenarios/question/native.cy.spec.js:
```diff
@@ -208,4 +208,54 @@ describe("scenarios > question > native", () => {
+      cy.log("**Assert that NULL value is present**");
+      cy.findByText(`V ${filter.toLowerCase()} This has a value`);
+      cy.findByText("No results!").should("not.exist");
+      cy.findByText(/null/i);
```
Yes, there's 2 rows - 1 with a string and 1 with `null` - so the "Is not" and "Does not contain" should return 1 result.

Commit:
Refactor the test

- make the final assertion using "count of rows
- apply different filters to the same question instead of creating 2 separate questions

nemanjaglumac commented:
@flamber - I have updated the test according to your suggestions. Thank you.

One you give it a final review and approval, I'll amend and remove `[ci skip]` and let it run in the CI.

flamber reviewed (approved):
Go go unskippy

sbelak commented:
This repro actually finds a different bug -- the FE gives you "no results" if the dataset returned is `[[null]]`.
